frontend/src/components/SeedlotRegistrationSteps/CollectionStep/index.tsx

```diff
@@ -0,0 +1,320 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
Remove this eslint-disable, I think this was left here on a first version of the changes, it's not necessary anymore.

Reply:
Fixed. Please review it again.